lib/beaker-hostgenerator/data.rb

```diff
@@ -939,7 +995,8 @@ def osinfo
         },
         'windows2012-64' => {
           :general => {
-            'platform' => 'windows-2012-64',
+            'platform'           => 'windows-2012-64',
+            'packaging_platform' => 'windows-2012-x64',
```

Comment:
Every windows entry can be updated to include `'packaging_platform' => 'windows-2012-x64',` or `'packaging_platform' => 'windows-2012-x84',`. The `2012` is arbitrary. We just need a value there for the automation. In reality, the windows msi are not version specific.